contracts/IERC721.sol

```diff
@@ -5,7 +5,9 @@ pragma solidity ^0.8.0;
  * See https://ethereum.org/en/developers/docs/standards/tokens/erc-721/#events for more information.
  */
 interface IERC721Events {
-    event ApprovalForAll(address indexed owner, address indexed operator, bool approved);
+    event ERC721Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
+    event ERC721Approval(address indexed owner, address indexed approved, uint256 indexed tokenId);
+    event ERC721ApprovalForAll(address indexed owner, address indexed operator, bool approved);
```

Comment:
Also, take a look at the section https://github.com/hashgraph/hedera-forking?tab=readme-ov-file#non-fungible-tokens. These tables are auto-generated using the Foundry `IERC721` interface. Let's update the NatSpec docs on our `IERC721` so the docs come from it.

Reply:
See at the end of section https://github.com/hashgraph/hedera-forking?tab=readme-ov-file#build on how to update the `README` from the interfaces.